src/server.js

```diff
@@ -102,12 +104,12 @@ async function startServer (options) {
 
   server.makeOp = async function (bot) {
     wrap.writeServer(`op ${bot.username}`)
-    await sleep(50)
+    await awaitServerMessage(bot, 'chat.type.admin')
   }
 
   server.teleport = async function (bot, pos) {
     wrap.writeServer(`tp ${bot.username} ${pos.x} ${pos.y} ${pos.z}`)
-    await sleep(1000)
+    await bot.waitForChunksToLoad()
```

Comment:
On this one, you should wait for a teleport package before waiting for chunks to load. Otherwise, the bot will wait for chunks at its current location instead of the target location.

Reply:
Fixed, now await "forcedMove" which: "Fires when the bot is force moved by the server (teleport, spawning, ...)."